decidim-core/app/assets/javascripts/decidim/editor.js.es6

```diff
@@ -0,0 +1,46 @@
+//= require quill.min
+//= require_self
+
+$(document).on('turbolinks:load', () => {
+  $(() => {
```

Comment:
Well, we have feature test to cover this. I'm not sure if unit tests give us much value right now because we may re-implement this using a React Component. This will introduce a new API for testing as well. What do you think?